nav2_system_tests/src/recoveries/spin_recovery_tester.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
You may want to add your own copyright below this 

 // Copyright (c) 2020 XYZ